app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/RepresentedPersonDetails.scala

```diff
@@ -16,24 +16,53 @@
 
 package uk.gov.hmrc.cgtpropertydisposals.models.des.returns
 
+import java.time.LocalDate
+
 import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.cgtpropertydisposals.models.address.Address
 import uk.gov.hmrc.cgtpropertydisposals.models.des.AddressDetails
+import uk.gov.hmrc.cgtpropertydisposals.models.returns.RepresenteeDetails
 
 final case class RepresentedPersonDetails(
   capacitorPersonalRep: String,
   firstName: String,
   lastName: String,
-  idType: String, //TODO: "ZCGT","NINO","UTR","TRN"
+  idType: String,
   idValue: String,
   dateOfBirth: Option[String],
-  dateOfDeath: Option[String],
+  dateOfDeath: Option[LocalDate],
   trustCessationDate: Option[String],
   trustTerminationDate: Option[String],
   addressDetails: Option[AddressDetails],
   email: Option[String]
 )
 
 object RepresentedPersonDetails {
+
+  def apply(representeeDetails: RepresenteeDetails): RepresentedPersonDetails = {
+    val idTypeWithValue = representeeDetails.id match {
+      case Left(sautr)          => "UTR"  -> sautr.value
+      case Right(Left(nino))    => "NINO" -> nino.value
+      case Right(Right(cgtRef)) => "ZCGT" -> cgtRef.value
```

Comment:
are you missing `TRN` here ?

Reply:
we don't allow trusts to be represented by capacitors or personal representatives, so that wasn't included in `RepresenteeRefrenceId`. TRN in the DES spec because it was drawn up to future proof against things we didn't know at the time